includes/options/views/class-amp-setup-wizard-submenu-page.php

```diff
@@ -74,13 +109,23 @@ public function enqueue_assets( $hook_suffix ) {
 			true
 		);
 
+		wp_enqueue_style(
+			self::JS_HANDLE,
+			amp_get_asset_url( 'css/amp-setup-compiled.css' ),
+			[],
+			AMP__VERSION
+		);
+
```

Comment:
Looks like you forgot to enqueue the RTL version here.

```suggestion
		wp_styles()->add_data( self::JS_HANDLE, 'rtl', 'replace' );
```